pkg/bff/config/config.go

```diff
@@ -81,6 +82,14 @@ type MTLSConfig struct {
 	PoolPath string `split_words:"true"`
 }
 
+// EmailConfig defines how emails are sent from the BFF.
+type EmailConfig struct {
+	ServiceEmail   string `envconfig:"GDS_BFF_SERVICE_EMAIL" default:"TRISA Directory Service <admin@vaspdirectory.net>"`
+	SendGridAPIKey string `envconfig:"SENDGRID_API_KEY" required:"false"`
```

Comment:
This may just be a copy-and-paste thing from the GDS config, but I'm noting here that `required` is `false` -- but is not validated when `Testing` is also false. I'll propose something below, but if it also needs to be updated in the GDS email manager, would you mind adding it there too?